frontend/src/views/MovieSessionDetailsScreen.vue

```diff
@@ -68,7 +68,7 @@ export default {
     async fetchMovieSession (id) {
       try {
         this.loading = true;
-        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}`, {
+        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}/`, {
```

Comment:
Are you sure you can get movie data from this endpoint? Add a screenshot of UI and developer tools when you want to see specific movie details
Check please if you fixed all endpoints